modules/curl/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://github.com/curl/curl",
+    "maintainers": [
+        {
+            "email": "julian.amann@tum.de",
```

Comment:
have you gotten Julian's consent? :)

Reply:
Seems that maintaining curl is not a no brainer. Therefore, I would opt-out here.